classes/game.js

```diff
@@ -5,30 +5,54 @@ class Game{
         this.turn = data.turn;
         this.board = new Board(data.board);
         this.me = new Snake(data.you);
+				this.test = data;
     }
     //todo:
     findFood(){
-			let foodx = request.body.board.food[0].x
-			let foody = request.body.board.food[0].y
-			
-			let locx = request.body.you.body[0].x
-			let locy = request.body.you.body[0].y
-			
-			if(locx < foodx){
-				data.move = 'right';
-			}else if(locx > foodx){
-				data.move = 'left';
-			}else if(locy < foody){
-				data.move = 'down';
-			}else if(locy > foody){
-				data.move = 'up';
+			let foodLength = Object.keys(this.test.board.food).length;
+			let foodArr = this.test.board.food;
+			
+			let closex = 0;
+			let closey = 0;
+			let foodDistanceSquare = 99999999999999;
+			
+			let locx = this.test.you.body[0].x
```

Comment:
`this.me` references your snake (set in the constructor)
so you can just do `this.me.head.x` and `this.me.head.y` to get the head coordinates

Naming is also a bit hard to tell what you are using it for, I would try using a name that describes its purpose. If you are just using it as a reference point, maybe use something like `headX` and `myHeadX` so you know what it means later on in the code without having to go back and look at what it is. pascal case is good too, locx looks like one word